.github/workflows/ci.yml

```diff
@@ -1,6 +1,8 @@
 name: "ci"
 
-on: [push]
+on:
+  push:
+  workflow_dispatch:
```

Comment:
What does this do?

Reply:
That allows one to run the workflow manually. See [here](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/).